client/src/locales/en.json

```diff
@@ -68,6 +68,21 @@
   "authorization.auth-box.sign-in": "Sign in with:",
   "authorization.auth-box.cta": "Try it out:",
   "authorization.auth-box.demo-button": "Live demo",
+  "authorization.actions.logout-fail": "Oops, we're sorry, logout failed.",
```

Comment:
Stop using this Oops sorry. 
Just puy normal message like. `Logout failed. `
They're already removed from every message.